lib/cdo/slack.rb

```diff
@@ -19,6 +19,18 @@ class Slack
 
   SLACK_TOKEN = CDO.slack_token.freeze
 
+  # @param email [String] The email of the Slack user.
+  # @raise [ArgumentError] If the email does not correspond to a Slack user.
+  # @return [nil | String] The mention name for the Slack user.
+  def self.mention_name(email)
```

Comment:
Also adding a WARNING to the method comment noting that the mention character `@` is not included.